notifications_utils/postal_address.py

```diff
@@ -0,0 +1,122 @@
+from notifications_utils.countries import UK, Country, CountryNotFoundError
+from notifications_utils.countries.data import Postage
+from notifications_utils.formatters import (
+    normalise_lines,
+    remove_whitespace_before_punctuation,
+)
+from notifications_utils.recipients import (
+    is_a_real_uk_postcode,
+    first_column_headings,
+    format_postcode_for_printing,
+)
+
+
+address_lines_1_to_6_and_postcode = [
+    # The API only accepts snake_case placeholders
+    line.replace(' ', '_') for line in first_column_headings['letter']
+]
+address_lines_1_to_6 = address_lines_1_to_6_and_postcode[:-1]
+address_line_7 = 'address_line_7'
+
+
+class PostalAddress():
+
+    MIN_LINES = 3
+    MAX_LINES = 7
+
+    def __init__(self, raw_address):
+        self.raw_address = raw_address
+
+    def __bool__(self):
+        return bool(self.normalised)
+
+    def __repr__(self):
+        return f'{self.__class__.__name__}({repr(self.raw_address)})'
+
+    @classmethod
+    def from_personalisation(cls, personalisation_dict):
```

Comment:
Short for class. The first argument of a `classmethod` is always a the class itself. But since `class` is a reserved keyword in Python the convention is to use `cls` instead. See https://docs.python.org/3/library/functions.html#classmethod

Reply:
Cool, thanks a lot! 👍